### PR TITLE
feat: add bee open API versions info to the health endpoint

### PR DESCRIPTION
**Pull request by mrekucci**
This change is [Reviewable](https://reviewable.io/reviews/ethersphere/bee/2638)

### CODE REVIEW

**Comment by mrekucci** [comment]
@AuHau

**Review by notanatol — APPROVED** [review]

**Comment by AuHau** [comment]
WOW! What a speed! Thanks a lot @mrekucci 🙏

**Review by acud — APPROVED** [review]
nice!

**Comment by acud** [comment]
my only question would be - what would the endpoint give out when you just use `go build cmd/bee` and not `make binary`? and would it make the api panic or the libraries that read it break or throw an exception while parsing? maybe it would be beneficial to set the fallback to semver `0.0.0` so that parsing succeeds and that the version in this case is always _below_ some version that is expected, so that the libraries can issue the right warnings to the users. just an idea

File: pkg/api/version.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Copyright 2021 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package api
+
+// Version is set in the build process.
+var Version string
```
Comment (acud) [inline pkg/api/version.go:8]: it is actually not needed to make the member public. the build tags can also set a package scoped variable.

File: pkg/debugapi/version.go [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// Copyright 2021 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package debugapi
+
+// Version is set in the build process.
+var Version string
```
Comment (acud) [inline pkg/debugapi/version.go:8]: here too

File: pkg/api/version.go (hunk 1 above)
Comment (mrekucci) [reply pkg/api/version.go:8]: It is actually used it in the `debugapi` package so it must be exported.

File: pkg/debugapi/version.go (hunk 2 above)
Comment (mrekucci) [reply pkg/debugapi/version.go:8]: This doesn't need to but just for the symmetry with the `api` package, I'd also export it.

**Comment by mrekucci** [comment]
> my only question would be - what would the endpoint give out when you just use `go build cmd/bee` and not `make binary`? and would it make the api panic or the libraries that read it break or throw an exception while parsing? maybe it would be beneficial to set the fallback to semver `0.0.0` so that parsing succeeds and that the version in this case is always _below_ some version that is expected, so that the libraries can issue the right warnings to the users. just an idea

It'll return `{"status":"ok","version":"1.3.0-dev","apiVersion":"","debugApiVersion":""}`. We can do `0.0.0` as a fallback.

**Pull request merged**